src/useTracking.js

```diff
@@ -0,0 +1,24 @@
+import merge from 'deepmerge';
+import { useContext } from 'react';
+import { ReactTrackingContext } from './withTrackingComponentDecorator';
+
+export default function useTracking() {
+  const trackingContext = useContext(ReactTrackingContext);
+
+  if (!(trackingContext && trackingContext.tracking)) {
+    throw new Error(
+      'Attempting to call `useTracking` ' +
+        'without a ReactTrackingContext present. Did you forget to wrap the top of ' +
+        'your component tree with `track`?'
```

Comment:
very minor. Why not use template strings here?

Reply:
Mostly for indentation reasons, but no real preference